acquia_lift.admin.wizard.inc

```diff
@@ -605,6 +676,28 @@ function acquia_lift_personalize_campaign_wizard_review_alter(&$form, &$form_sta
     $form['review']['warnings_column']['warnings']['#type'] = 'container';
     $form['review']['warnings_column']['warnings']['#theme'] = 'acquia_lift_card';
   }
+  $form['#submit'] = array(
```

Comment:
This is going to be one of those merge things... this submit isn't needed anymore because the review subform implements the standard submit callback now in the workflow branch.
https://github.com/acquia/acquia_lift/blob/workflow/acquia_lift.admin.wizard.inc#L610